src/user/user.controller.spec.ts

```diff
@@ -3,7 +3,7 @@ import { Test, TestingModule } from "@nestjs/testing";
 import { UserController } from "./user.controller";
 import { UserModule } from "./user.module";
 import { expect } from "chai";
-import { DatabaseModule } from "../data/database.module";
+import { DatabaseModule } from "../database/database.module";
```

Comment:
Also absolute paths - I think the trick with this is that you need to tell jest about it too.

I personally don't have an opinion preference on something like `src/...` vs something like `~/...` `~/src/...` or whatever, but I don't like the way `../` adds an extra chore every time you move a file and a tiny bit of extra mental overhead every time you include one. It's not like either is in any way a problem, but tiny little things add up.

Reply:
I don't think absolute paths can work, can they?

Reply:
Yeah, there are a few different ways, although I have to admit I started, failed and gave up first time I looked at it.